lib/src/ast/css/media_query.dart

```diff
@@ -54,8 +54,8 @@ class CssMediaQuery {
     var theirType = other.type?.toLowerCase();
 
     if (ourType == null && theirType == null) {
-      return MediaQuerySuccessfulMergeResult._(CssMediaQuery.condition(
-          this.features.toList()..addAll(other.features)));
+      return MediaQuerySuccessfulMergeResult._(
+          CssMediaQuery.condition([...this.features, ...other.features]));
```

Comment:
MRW seeing JavaScript syntax array literals with destructing in Dart

![its-a-unix-system](https://user-images.githubusercontent.com/838736/57888358-d28ab580-77e6-11e9-8fc9-c9ac624b1780.gif)

Reply:
fwiw this is not destructuring though, this is spread. Dart does not have destructuring.